app/models/site.rb

```diff
@@ -2,4 +2,8 @@
 
 class Site < ApplicationRecord
   has_many :planning_applications, dependent: :restrict_with_exception
+
+  def full_address
+    "#{address_1}, #{town}, #{postcode}"
+  end
```

Comment:
Did we try using a view helper method for this? I'd prefer this string formatting to stay in our view layer.